verdigado_template_default/views/sepa_invoice.xml

```diff
@@ -41,6 +41,7 @@
             </t>
         </xpath>
         <!-- we replace the whole template id sdd_report_invoice_document... -->
+        <!--
         <xpath expr="//p[@t-if='o.sdd_mandate_id']" position="replace">
```

Comment:
diesen Teil finde ich in keiner View, wo kommt das her?

Reply:
Das kommt aus Odoo-Enterprise, Modul `account_sepa_direct_debit`.

Reply:
Für das Thema Zeiterfassung sind wir auf der Community-Edition. Für das interne Rechnungswesen sind wir auf Enterprise (war zuerst da). `verdigado_template_default` wird nur dort eingesetzt.

Reply:
ah, dann mache ich meine Änderungen oben rückgängig und exclude das Modul was die CI angeht?

Reply:
Achja, soweit hab ich nicht gedacht bzw. es nicht verstanden. Klar, dann musst Du das Modul excluden.